hudi-common/src/main/java/org/apache/hudi/common/model/debezium/AbstractDebeziumAvroPayload.java

```diff
@@ -76,7 +77,12 @@ private Option<IndexedRecord> handleDeleteOperation(IndexedRecord insertRecord)
     boolean delete = false;
     if (insertRecord instanceof GenericRecord) {
       GenericRecord record = (GenericRecord) insertRecord;
-      Object value = record.get(DebeziumConstants.FLATTENED_OP_COL_NAME);
+      Object value;
+      try {
+        value = record.get(DebeziumConstants.FLATTENED_OP_COL_NAME);
+      } catch (AvroRuntimeException e) {
```

Comment:
maybe wrap a static method that have a param to indicate whether return null if key is not exists.

Reply:
done